stan/math/prim/scal/prob/logistic_log.hpp

```diff
@@ -25,7 +25,6 @@ namespace stan {
   namespace math {
 
     // Logistic(y|mu, sigma)    [sigma > 0]
-    // FIXME: document
```

Comment:
You removed some of the signature doc, but not this one.  These should really be part of actual function doc rather than line comments.